en/deploy-on-azure-aks.md

```diff
@@ -0,0 +1,552 @@
+    --resource-group ${resourceGroup} \
+    --node-vm-size ${nodeType} \
+    --zones 1 2 3 \
+    --aks-custom-headers EnableAzureDiskFileCSIDriver=true \
+    --node-count 3 \
+    --labels dedicated=tikv \
+    --node-taints dedicated=tikv:NoSchedule \
+    --enable-ultra-ssd
+```
+
+By default, only two TiDB nodes are required, so you can set the `--node-count` of the `tidb` node pool to `2`. You can scale out this node pool any time if necessary.
+
+Each command above might take several minutes. For more cluster configuration, refer to [`az aks` documentation](https://docs.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az_aks_create) and [`az aks nodepool` documentation](https://docs.microsoft.com/en-us/cli/azure/aks/nodepool?view=azure-cli-latest)
+
+Azure AKS cluster deploy nodes across multiple zones using "best effort zone balance", if you need "strict zone balance" (not supported in AKS now), consider deploy a node pool in each zone. For example:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+# create tikv node pool in zone 1
+az aks nodepool add --name tikv1 \
+    --cluster-name ${clusterName} \
+    --resource-group ${resourceGroup} \
+    --node-vm-size ${nodeType} \
+    --zones 1 \
+    --aks-custom-headers EnableAzureDiskFileCSIDriver=true \
+    --node-count 1 \
+    --labels dedicated=tikv \
+    --node-taints dedicated=tikv:NoSchedule \
+    --enable-ultra-ssd
+
+# create tikv node pool in zone 2
+az aks nodepool add --name tikv2 \
+    --cluster-name ${clusterName} \
+    --resource-group ${resourceGroup} \
+    --node-vm-size ${nodeType} \
+    --zones 2 \
+    --aks-custom-headers EnableAzureDiskFileCSIDriver=true \
+    --node-count 1 \
+    --labels dedicated=tikv \
+    --node-taints dedicated=tikv:NoSchedule \
+    --enable-ultra-ssd
+
+# create tikv node pool in zone 3
+az aks nodepool add --name tikv3 \
+    --cluster-name ${clusterName} \
+    --resource-group ${resourceGroup} \
+    --node-vm-size ${nodeType} \
+    --zones 3 \
+    --aks-custom-headers EnableAzureDiskFileCSIDriver=true \
+    --node-count 1 \
+    --labels dedicated=tikv \
+    --node-taints dedicated=tikv:NoSchedule \
+    --enable-ultra-ssd
+```
+
+> **Warning:**
+>
+> About node pool scale in:
+>
+> * You can manually scale an AKS cluster to run a different number of nodes. When you scale down, nodes are carefully [cordoned and drained](https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/) to minimize disruption to running applications, refer [Scale the node count in an Azure Kubernetes Service (AKS) cluster](https://docs.microsoft.com/en-us/azure/aks/scale-cluster).
+> * If you want to use the cluster autoscaler, you can [Change the cluster autoscaler settings](https://docs.microsoft.com/en-us/azure/aks/cluster-autoscaler#change-the-cluster-autoscaler-settings) and refer [Enable TidbCluster Auto-scaling](https://docs.pingcap.com/tidb-in-kubernetes/stable/enable-tidb-cluster-auto-scaling)
```

Comment:
Please remove the reference for `Enable TidbCluster Auto-scaling`, it's not ready yet.

Reply:
done